apps/remix-ide/src/app/files/fileManager.ts

```diff
@@ -255,13 +255,13 @@ class FileManager extends Plugin {
    * @param {string} path path of the file
    * @returns {string} content of the file
    */
-  async readFile(path) {
+  async readFile(path, isBuffer?) {
```

Comment:
if we start doing this we should maybe just add encoding as a param, which is the standard fs way. so we can support everything.